src/markdown_generators_test.js

```diff
@@ -285,7 +285,7 @@ describe("generateCourseSectionFrontMatter", () => {
           "Syllabus",
           "syllabus",
           null,
-          false,
+          true,
           false,
           false,
           10,
```

Comment:
would it be good to add a check that a non-root page without `list_in_left_nav` doesn't get a menu entry?

Reply:
tests added